autoconnect/autoconnect-ws/autoconnect-ws-sm/src/identified/on_client_msg.rs

```diff
@@ -330,4 +335,42 @@ impl WebPushClient {
             Ok(vec![])
         }
     }
+
+    /// Update minimum urgency for the user and the flag
+    ///
+    /// If the new urgency is lower than the previous one,
+    /// We check pending messages, to send messages that were
+    /// retained because of their urgency
+    async fn change_min_urgency(
+        &mut self,
+        new_min: Urgency,
+    ) -> Result<Vec<ServerMessage>, SMError> {
+        // Change the min urgency
+        self.flags.min_urgency = new_min;
+
+        let status = if let Some(mut user) = self.app_state.db.get_user(&self.uaid).await? {
+            let current_urgency = user.urgency.unwrap_or(Urgency::VeryLow);
```

Comment:
Wouldn't the default urgency be `Urgency::Normal`?

I presume that any message without an explicit urgency flag would be prioritized over any message that had specified a lower than normal urgency.

Reply:
I've added a comment about the fallback. It uses VeryLow as a default because if the user doesn't set a minimum, it means they want all messages